src/sdk/model/vdc/__json__/vm-affinity-rule-list-json.ts

```diff
@@ -0,0 +1,5 @@
+import { VmAffinityRuleJson } from './vm-affinity-rule-json';
+
+export interface VmAffinityRuleListJson {
```

Comment:
this still doesn't appear to be used anywhere?

Reply:
Oh i remove the class but not the JSON descriptor... doing it now.

Reply:
Done.